src/three/TilesRenderer.js

```diff
@@ -172,24 +181,28 @@ export class TilesRenderer extends TilesRendererBase {
 
 	raycast( raycaster, intersects ) {
 
-		if ( ! this.root ) {
+		if ( this.optimizeRaycast ) {
 
-			return;
+			if ( ! this.root ) {
 
-		}
+				return;
 
-		if ( raycaster.firstHitOnly ) {
+			}
```

Comment:
I don't think we should have to adjust the `TilesRenderer.raycast` function. Instead we should adjust [TilesGroup.raycast](https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/master/src/three/TilesGroup.js#L16-L20) to check if the tiles renderer (which it already has a reference to) has the tiles override flag is set to true. That way `TilesRenderer.raycast` will always perform the optimized version. Perhaps we can rename this function and expose it in a future release so it can be usable independent of three.js' Raycaster mechanism.

Reply:
Also resolved.